reporting/reports/data/system-history

```diff
@@ -89,7 +94,7 @@ sql:
 				when summary like 'unsubscribed from channel %' then details
 				when summary like 'Updated system release %' then substr(summary, 24)
 			end
-		from rhnserverhistory
+		from rhnserverhistory left outer join rhnserver on rhnserverhistory.server_id = rhnserver.id
```

Comment:
Same as above.

Reply:
I have changed the join to inner join in the second commit.